interfaceProject/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace interfaceProject
```

Comment:
I can see that you have the right idea, but none of you classes implement an interface.